src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -63,6 +64,41 @@ def create(self, name: str, service_spec: Dict[str, Any]):
                 f'Error creating volume {name} with placement {str(service_spec)}: {err}')
         return f'Volume {name} created successfully'
 
+    @EndpointDoc("Delete CephFS Volume",
+                 parameters={
+                     'name': (str, 'File System Name'),
+                 })
+    @allow_empty_body
+    @Endpoint('DELETE')
+    @DeletePermission
+    def remove(self, name):
+        error_code, _, err = mgr.remote('volumes', '_cmd_fs_volume_rm', None,
+                                        {'vol_name': name,
+                                         'yes-i-really-mean-it': "--yes-i-really-mean-it"})
+        if error_code != 0:
+            raise DashboardException(
+                msg=f'Error deleting volume {name}: {err}',
+                component='cephfs')
+        return f'Volume {name} deleted successfully'
+
+    @EndpointDoc("Rename CephFS Volume",
+                 parameters={
+                     'name': (str, 'Existing FS Name'),
+                     'new_name': (str, 'New FS Name'),
+                 })
+    @Endpoint('PUT')
```

Comment:
an `empty` space is needed before new method, otherwise lint would fail
```suggestion
    
    @Endpoint('PUT')
```